packages/replay-next/src/suspense/ObjectPreviews.ts

```diff
@@ -1,288 +1,87 @@
-import { Object, ObjectId, PauseId, Value as ProtocolValue } from "@replayio/protocol";
 import {
-  Deferred,
-  Record,
-  ResolvedRecord,
-  createDeferred,
-  createPendingRecord,
-  createResolvedRecord,
-  isPendingRecord,
-  isRejectedRecord,
-  isResolvedRecord,
-  updateRecordToRejected,
-  updateRecordToResolved,
-} from "suspense";
+  Object,
+  ObjectId,
+  ObjectPreviewLevel,
+  PauseId,
+  Value as ProtocolValue,
+} from "@replayio/protocol";
+import { Cache, createCache } from "suspense";
 
 import { ReplayClientInterface } from "../../../shared/client/types";
-import { createFetchAsyncFromFetchSuspense } from "../utils/suspense";
 import { cachePauseData } from "./PauseCache";
 
-type ObjectMap = Map<ObjectId, Object>;
-type RecordMap = Map<ObjectId, Record<Object>>;
-type PropertyRecordMap = Map<string, Record<ProtocolValue>>;
+export const objectCache: Cache<
```

Comment:
:speech_balloon: Had to go off and look at the final version of this file separate from the diff, but yeah, that _does_ look nice and compact!

Reply:
Thanks! 🙇🏼 

I'm pretty happy with how this specific cache turned out. Given that it was the most complex of the "simple" caches, I think it's a good signal for replacing the other `createGenericCache` instances.

Will be a bit more work to replace the interval and streaming cache types though.